### PR TITLE
test(flow-item, panel): add tests for internal property.

### PR DESCRIPTION
**Pull request by driskull**
**Related Issue:** #7930

## Summary

- Adds stories for internal collapseDirection property.

### CODE REVIEW

File: packages/calcite-components/src/components/flow-item/flow-item.stories.ts [hunk 1]
```diff
@@ -159,6 +167,10 @@ export const collapsed_TestOnly = (): string => html`
   <calcite-flow-item collapsed collapsible closable> Hello World! </calcite-flow-item>
 `;
 
+export const collapsedInverted_TestOnly = (): string => html`
```
Comment (jcfranco) [inline packages/calcite-components/src/components/flow-item/flow-item.stories.ts:170]: `collapsedInverted_TestOnly` ➡️ `collapseInverted_TestOnly`

**Review by jcfranco — COMMENTED** [review]
If this prop only affects the icon and not the way the panel expands, for consistency, could we rename it to follow something similar to `flipRtl`?

**Comment by driskull** [comment]
> If this prop only affects the icon and not the way the panel expands, for consistency, could we rename it to follow something similar to flipRtl?

@jcfranco I had considered `flipRTL` but it actually doesn't follow our naming standard. Ideally, it would be `rtl-flipped` to be more inline with how HTML names attributes right? Do we want to consider updating that property to be more inline with our other props?

**Review by alisonailea — APPROVED** [review]

**Comment by macandcheese** [comment]
Since ‘collapseInverted’ doesn’t really explain what the default direction is, maybe ‘collapseDirection’ top / bottom with a meaningful default is easier to understand? 


It does seem like it would maybe be confusing or lead to misuse as an option in Block, Accordion, etc, so would this only be available in Panel?

**Comment by driskull** [comment]
> Since ‘collapseInverted’ doesn’t really explain what the default direction is, maybe ‘collapseDirection’ top / bottom with a meaningful default is easier to understand?

I agree. Maybe you me and @jcfranco can figure out a name? I can also just make this internal for now.

> It does seem like it would maybe be confusing or lead to misuse as an option in Block, Accordion, etc, so would this only be available in Panel?

I think it could be useful in anything that uses expand arrows to indicate expand direction. Currently, we assume that accordion and block open below but if the component is anchored at the bottom of something then that isn't the case anymore.

**Comment by jcfranco** [comment]
> Ideally, it would be rtl-flipped to be more inline with how HTML names attributes right? Do we want to consider updating that property to be more inline with our other props?

Good point. I don't think we focus on this explicitly in our conventions. From what I recall, we've suggested concise adjectives for boolean names. We could update the conventions and do another audit on boolean names.

FWIW, I was more suggesting to use `flip` in the name. 😅

> Since ‘collapseInverted’ doesn’t really explain what the default direction is, maybe ‘collapseDirection’ top / bottom with a meaningful default is easier to understand?

Agreed. Would we ever have other directions? 

I'm fine with keeping it internal while we hash this out to land the increased test coverage.

**Review by jcfranco — APPROVED** [review]
Aside from the naming discussion, LGTM!

Good to merge as is if the prop is made internal.

**Comment by macandcheese** [comment]
> I think it could be useful in anything that uses expand arrows to indicate expand direction. Currently, we assume that accordion and block open below but if the component is anchored at the bottom of something then that isn't the case anymore.

Yeah, although I think flipping the direction of the icon is unneeded because the area that expands is still opening in the same space / direction - the containing element just gets taller. A "real flip" IMO would be the content area opening on the "other side" of the element.

**Comment by driskull** [comment]
Reverted to previous name and `@internal` state.

**Comment by driskull** [comment]
@ashetland can you review screenshot tests?

**Pull request merged**